.github/workflows/tos.yaml

```diff
@@ -54,15 +55,15 @@ jobs:
 
   parse-issue:
     needs: [ react-to-new-issue, is-revalidation ]
-    if: always() && github.event.issue.state == 'open'
+    if: failure() == false && github.event.issue.state == 'open'
```

Comment:
Using `failure() == false` instead of `always()` makes sure that the issue is parsed only if one of the needs is successful and the other is skipped. If one of the needs is failed (exist 1 for example) `parse-issue` is not executed, and neither is the validation.